README.md

```diff
@@ -1,42 +1,55 @@
+
+If this the first time you're contributing, click **Edit the file in your fork of this project** (pencil icon) on the top right corner to create a fork. 
+
+### Creating Pull Requests
+
+1. When you're ready to submit your changes, add a descriptive title and comments to summarize the changes made.
+2. Select **Create a new branch for this commit and start a pull request**.
+3. Check the **Propose file change** button.
+4. Scroll down to compare changes with the original document.
+5. Select **Create pull request**. 
+
+### Using Labels
+
+Labels are used to track the lifecycle and status of a pull request. Using the correct labels helps with managing workflows and ensuring that content is edited, merged, released etc at the correct time. Take a look at the [Labels](https://developers.mattermost.com/contribute/getting-started/labels/) page for information about how and when to use which labels.
 
-**How to create Pull Requests to edit or create Mattermost documentation (by community members):**
+### Commenting on Pull Requests
 
-1. Sign the Contributor License Agreement (see instructions in the next section).
-3. On the Mattermost Documentation page that you want to edit, click the GitHub icon on the upper right corner that says "Edit".
-4. Click "Edit the file in your fork of this project" (pencil icon) on the upper right corner.
-5. After making changes, check the "Propose file change" button.
-6. Compare changes with the original document.
-7. Click "Create a Pull Request". Make sure that the Pull Request has a descriptive title and add comments to briefly tell others what you have worked on (optional).
+Once a pull request is submitted, multiple committers may comment on it and provide edits or suggestions which you can commit directly. You can also add line comments. Take a look at [Commenting on pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) for more details.
 
-**Signing CLA:**
+### Reviewing Pull Requests
 
-Please read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it (at the bottom of the page), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
+Once a pull request has been submitted and the correct label assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers. 
 
-**Additional process for Core Committers for Doc Repo:**
+Once the review process is complete and depending on the type of issue it is (e.g., a typo fix vs. a new feature), the change is either merged into master and pushed immediately or merged into the release branch and pushed in alignment with the release. The branch is then deleted. 
 
```

Comment:
Any merged PRs with a "Editor Review” or "Reviews Complete" label will be processed by the editor reviewer to ensure the documentation is correctly formatted at https://docs.mattermost.com/.